docs/content/developer/getting-started/install-iota.mdx

```diff
@@ -73,18 +73,24 @@ Once you have downloaded and extracted the `.tgz` file that matches your operati
 and
 install the necessary binaries. You should start with the main IOTA binary.
 
-1. iota-`<OS>`-`<ARCHITECTURE>`: Main IOTA binary.
-2. iota-faucet-`<OS>`-`<ARCHITECTURE>`: Local faucet to mint coins on a local network.
-3. iota-indexer-`<OS>`-`<ARCHITECTURE>`: An indexer for a local IOTA network.
-4. iota-node-`<OS>`-`<ARCHITECTURE>`: Run a local node.
-5. iota-tool-`<OS>`-`<ARCHITECTURE>`: Provides utilities for IOTA.
+1. iota: Main IOTA binary.
+2. iota-faucet: Local faucet to mint coins on a local network.
+3. iota-indexer`: An indexer for a local IOTA network.
+4. iota-node: Run a local node.
+5. iota-tool: Provides utilities for IOTA.
 
 :::note
 
 This process may take several minutes to download and install; this depends on your network and PC specifications.
 
 :::
 
+Add the folder containing the extracted files to your PATH variable. To do so, you can update your `~/.zshrc` or `~/.bashrc` to include the location of the IOTA binaries. If using the suggested location, you type export PATH=$PATH:~/iota and press Enter.
+
+Start a new terminal session or type source ~/.zshrc (or .bashrc) to load the new PATH value.
+
+If you are running the binaries for the first time, you might receive an error from MacOS that prevents the binaries from running. If you receive this error, close the dialog and type xattr -d com.apple.quarantine ~/iota/* in your terminal and press Enter (be sure to adjust the path if different).
```

Comment:
```suggestion
If you are running the binaries for the first time, you might receive an error from MacOS that prevents the binaries from running. If you receive this error, close the dialog and type `xattr -d com.apple.quarantine ~/iota/*` in your terminal and press Enter (be sure to adjust the path if different).
```
Which error? Please add the specifics

Reply:
When you run the iota command the first time, it will show this. I also confirmed it from the Sui doc.

<img width="265" alt="Screenshot 2025-02-14 at 13 43 12" src="https://github.com/user-attachments/assets/b569fdc7-4fa3-4125-8c68-c4d0406c0859" />

<img width="647" alt="Screenshot 2025-02-14 at 13 47 28" src="https://github.com/user-attachments/assets/03d34843-634b-4d81-90e0-6584b45868d0" />


It will show this on Mac, and you need to go to privacy and security to allow it.

Reply:
I meant that please add the error message to the docs. Instead of saying an error, you say you may receive an error stating that `iota` cannot .... btw, we should add a checksum or smt so the users can verify the downloaded binaries

Reply:
@lucas-tortora I have added admonition. Please feel free to review. About checksum, do we have an official checksum for the binaries? Do you mean I should run something like `sha256sum iota` before running the binaries?